connectors/sources/abs.py

```diff
@@ -250,7 +250,7 @@ async def get_blob(self, container):
                     blob=blob, container_metadata=container["metadata"]
                 )
 
-    async def get_docs(self):
+    async def get_docs(self, **kwargs):
```

Comment:
Also had it like this way before. I thought it would be better for connectors which don't need filtering to keep it this way. But happy to change it to have it more explicit.

Reply:
I am thinking about the base class signature and documenting the behavior see https://github.com/elastic/connectors-python/blob/main/connectors/source.py#L194-L219

if calling `get_docs` has just that one optional argument and nothing else. I would set it as an explicit arg.

Reply:
Adjusted it